### PR TITLE
CSV inventory plugin

### PR DESCRIPTION
**Pull request by snopoke**
See docs and examples in diff for how it works and expected format.

Also see here for another example: https://docs.google.com/spreadsheets/d/1WlamplK6thSqbPt2fMWsoHkPwOan9kdzN63uwu_G7Zo/edit#gid=403315111

### CODE REVIEW

File: src/commcare_cloud/ansible/plugins/inventory/csv.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                    row_groups.append(current_group)
+                    current_group = []
+            else:
+                current_group.append(line)
+        if current_group:
+            row_groups.append(current_group)
+        return row_groups
+
+
+def conv_str2value(item_type, item):
```
Comment (snopoke) [inline src/commcare_cloud/ansible/plugins/inventory/csv.py]: The `ini` parser uses [ast.literal_eval](https://docs.python.org/2/library/ast.html#ast.literal_eval) to parse var values - I could switch to that but thought that being explicit was better. Open to opinions.

File: src/commcare_cloud/ansible/plugins/inventory/csv.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                item_type, name = key.split('.') if '.' in key else ('S', key)
+                name = name.split(' ')[1]
+                vars[name] = conv_str2value(item_type, val)
+        return vars
+
+    def _parse_groups(self, rows):
+        for row in rows:
+            group = row['group']
+            self.inventory.add_group(group)
+            var_name, type_, raw_val = row['var'], row['type'], row['val']
```
Comment (calellowitz) [inline src/commcare_cloud/ansible/plugins/inventory/csv.py]: nit: you use `item_type` above to refer to the same thing (which i generally like better than the trailing underscore). consistency would be nice.

**Comment by calellowitz** [comment]
That was supposed to be an approval not a change request

**Comment by emord** [comment]
I think the tests are failing due to docs not being made?

File: src/commcare_cloud/ansible/ansible.cfg [hunk 3]
```diff
@@ -11,5 +11,10 @@ fact_caching_timeout = 21600
 
 timeout = 30
 
+inventory_plugins = ./plugins/inventory
```
Comment (emord) [inline src/commcare_cloud/ansible/ansible.cfg]: For me this only works if i'm already in the `src/commcare_cloud/ansible` directory. I was trying to run `cchq icds-cas x` but this plugin would not be found

**Comment by emord** [comment]
fyi I added a commit to check for the new inventory paths

File: src/commcare_cloud/ansible/ansible.cfg (hunk 3 above)
Comment (snopoke) [reply src/commcare_cloud/ansible/ansible.cfg]: odd, it should be relative to this file (same as callback plugins)

Comment (snopoke) [reply src/commcare_cloud/ansible/ansible.cfg]: I this should fix it: https://github.com/dimagi/commcare-cloud/pull/2050/files#diff-9af49c7720ecc05ddbc9456b1c90b73bR10

**Pull request merged**